docs/getting-started-devs/integrations.md

```diff
@@ -135,7 +164,169 @@ nanoblock:<json>
+
+An example of a DNS SRV record for the subdomain `nano.example.com` using the default port of 443.
+
+```dns
+_nano_currency._tcp.example.com 86400 IN SRV 0 0 443 nano.example.com
+```
+
+#### Static self-hosting
+
+Users may choose to host their own Nano identifiers through a static file server.
+
+This may be done in the same way as a public static file server, by hosting the file `.well-known/nano-currency.json` from the root of the static file server, in the same format as described above, such as
+
+```json
+{
+    "names": [
+        {
+            "name": "user",
+            "address": "nano_3wm37qz19zhei7nzscjcopbrbnnachs4p1gnwo5oroi3qonw6inwgoeuufdp",
+            "expiry": null
+        },
+    ]
+}
+```
+
+#### Allowing access from JavaScript clients
+
+JavaScript Nano clients may be restricted by browser [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) policies that prevent them from accessing `/.well-known/nano-currency.json` on the user's domain. 
+
+When CORS prevents JS from loading a resource, the JS program sees it as a network failure identical to the resource not existing, so it is not possible for a pure-JS app to tell the user for certain that the failure was caused by a CORS issue. 
+
+JavaScript Nano clients that see network failures requesting `/.well-known/nano-currency.json` files may want to recommend to users that they check the CORS policy of their servers, e.g.:
+
+```
+$ curl -sI https://example.com/.well-known/nano-currency.json?names=user | grep -i ^Access-Control
+Access-Control-Allow-Origin: *
+```
+
+Services should ensure that their `/.well-known/nano-currency.json` is served with the HTTP header `Access-Control-Allow-Origin: *` to ensure it can be validated by pure JS clients running in modern browsers.
+
+#### Security constraints
+The `/.well-known/nano-currency.json` endpoint MUST NOT return any HTTP redirects.
+
+Clients MUST ignore any HTTP redirects given by the `/.well-known/nano-currency.json` endpoint.
+
+#### Re-assignment of Nano identifiers
+
+Nano identifier services SHOULD ONLY allow re-assignment of currently or previously registered identifiers to the current or original owner, even in the case of an expired registration. 
+
+For example, a Nano identifier service MAY allow the original owner of an expired registered identifier to re-register the identifier to the original or a new Nano public address by requiring authentication with the originally assigned address or an alternative authentication method.
+
+This restriction ensures that a new user cannot claim an expired identifier and receive payments that were intended for the original owner.
+
+### Client Use of Nano Identifiers
+
+Upon encountering a Nano identifier in the format `<local-part>@<domain>`, the client SHOULD perform an address lookup by sending a GET request to the endpoint `<domain>/.well-known/nano-currency.json?names=<local-part>`.
```

Comment:
preferably `@<local-part>@<domain>`, to make it less likely a user interprets alias as an ability to send funds to an email address (similarly, the beginning of the `Nano Internet Identifiers` section should clarify that)

additionally, it would be nice to spec `@<domain>` as a shorter alternative to writing `@_@<domain>` . some of the services already support the short version

Reply:
Yeah I'll make that change.